README.md

```diff
@@ -2,8 +2,147 @@
 
 [![Build Status](https://travis-ci.org/hmrc/file-transmission.svg)](https://travis-ci.org/hmrc/file-transmission) [ ![Download](https://api.bintray.com/packages/hmrc/releases/file-transmission/images/download.svg) ](https://bintray.com/hmrc/releases/file-transmission/_latestVersion)
 
-This is a placeholder README.md for a new repository
+Microservice that facilitates transmission of files between MDTP and MDG.
+
+Whenever service on MDTP platform wants to send bundle (batch) of files to MDG it will need to use this service. The file-transmission
+service notifies MDG that files prepared by service on MDTP are ready to be processed. Please note that file-transmission service 
+do not transfer file contents. The service just passes data about file location. The file content has to be provided by other service (eg. upscan)
+
+# Typical usage scenario
+
+- Consuming service requests upload of the file from user using `upscan` service
+- `Upscan` notifies consuming service that the file has been uploaded and is ready to be processed - giving consuming 
+service the URL that allows to download the file.
+- Consuming service ensures that all required files have been uploaded by the user
+- Consuming service uses file-transmission service to notify MDG that files are ready to be processes
+- MDG asynchronously uses the file
+
+## Service usage
+
+### Whitelisting
+
+In order to initiate an upload the consuming service must be whitelisted by file-transmission. See the 'Whitelisting client services' section further down in this document.
+
```

Comment:
White-listing does not allow the initiation of an upload in this case, but allows the service to access the interface for MDG to identify and locate hosted files (that it may or may not be able to download).